### DISCUSSION TITLE
Bookmarks bar is removed

### DISCUSSION THREAD

**Discussion by asawaronit60** [dailydotdev/daily#722] [Feature Requests]
Hii this is an awesome extension , just loved it .
But the problem is that it is removing my bookmarks bar in the chrome .

**Comment by idoshamun** [comment]
Hi @asawaronit60,

Please see this section on our docs:
https://docs.daily.dev/docs/getting-started/browser-extension-installation#my-chromes-bookmarks-bar-is-gone-how-can-i-bring-it-back
